packages/graphql/src/classes/subscription/Neo4jGraphQLSubscriptionsCDCEngine.ts

```diff
@@ -35,18 +37,22 @@ export class Neo4jGraphQLSubscriptionsCDCEngine implements Neo4jGraphQLSubscript
     private closed = false;
 
     private subscribeToLabels: string[] | undefined;
+    private filterGraphQLEvents: boolean;
 
     constructor({
         driver,
         pollTime = 1000,
         queryConfig,
+        filterGraphQLEvents = false,
```

Comment:
I think this nomenclature is awkward - when this is true it... Filters _out_ GraphQL events?

`onlyGraphQLEvents`?